src/conda/env_python3_base.yml

```diff
@@ -14,3 +14,6 @@ dependencies:
 - xarray=0.16
 - matplotlib=3.3
 - cartopy=0.18
+- numba = 0.51.2
+- networkx = 2.3
+- jupyterlab
```

Comment:
```suggestion
```

Per comments by @tsjackson-noaa and @wrongkindofdoctor.  I agree.

Reply:
@jkrasting, @tsjackson-noaa, @wrongkindofdoctor Jupyter removed from python3 base. 

While I was working on it, I noticed that the current version of git-intro documentation may not be ideal. It contains a lot of stuff, and I had difficulty finding the commands I needed (in the right order; e.g., `git commit -a` then `git push origin`). In the end, I went back to the older version @tsjackson-noaa wrote (last summer?) and found what I needed. 

I would like to suggest re-organizing/writing the git-intro so that users can follow it more easily (like @tsjackson-noaa's original version). Should I bring it up in this week's telecon or start an issue?